news/841.new.rst

```diff
@@ -0,0 +1 @@
+Add a hook for ``selectolax``, which has a hidden import.
```

Comment:
You are not solving hidden imports problem with this hook, though.

So the text should be something along the lines of `Add a hook for ``selectolax`` to collect its data files.`

Reply:
Addressed in 
- https://github.com/pyinstaller/pyinstaller-hooks-contrib/pull/841/commits/e7eeac877e5082a76c9225791d8e90a1bb8b01c4